1.0.0-beta4/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mono:3.12
+
+# Get build dependencies, download/build/install mono 4.1.0
+RUN apt-get update -qq \
+    && apt-get install -qqy git autoconf libtool automake build-essential mono-devel gettext unzip \
+    && git clone https://github.com/mono/mono.git \
+    && cd mono \
+    && git reset --hard 53dc56ee39a8e3b013231957aca4671b202c6410 \
+	&& ./autogen.sh --prefix="/usr/local" \
+    && make \
+    && make install \
+    && cd .. \
+    && rm mono -r
+
+# Install aspnet 1.0.0-beta4
+ENV DNX_FEED https://www.myget.org/F/aspnetrelease/api/v2
+ENV DNX_USER_HOME /opt/dnx
+
+RUN curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=release sh
```

Comment:
@northtyphoon This line will not always install the same version. The commit `release` can easily change (to something which is not beta4, and could break the depending product). Can you please find a better commit number or branch until `beta4` branch comes out?

We can always fix this after Apr 29 however if by any chance image gets rebuilt the day before and a different commit gets picked up and breaks the product, it'll be because we used a moving branch pointer rather than a commit ID or tag.